pkg/cmd/datastore/datastore.go

```diff
@@ -89,10 +89,7 @@ type Config struct {
 
 // RegisterDatastoreFlags adds datastore flags to a cobra command
 func RegisterDatastoreFlags(cmd *cobra.Command, opts *Config) {
-	engines := []string{}
-	for engine := range BuilderForEngine {
-		engines = append(engines, fmt.Sprintf("%q", engine))
-	}
+	engines := append([]string{}, datastore.Engines...)
 	sort.Strings(engines) // Sort for stability.
```

Comment:
```suggestion
	sort.Strings(datastore.Engines) // Sort for stability.
```

I don't think we really care if the original list gets sorted?

Reply:
I'd rather not change it, as its defined to be "static", for lack of a better term.